src/style.css

```diff
@@ -1,3 +1,71 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+header {
+  margin: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+nav {
+  margin: 0;
+}
+
+.nav-list {
+  height: 60px;
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+  margin-right: 20px;
+  height: 100%;
+  position: relative;
+}
+
+.nav__item:last-child {
+  margin: 0;
+}
+
+.logo-container {
+  height: 40px;
+  margin: 10px 0;
```

Comment:
Be consistent with your vertical margins. You've added a top margin here, but it's better to stick with either top or bottom throughout your CSS for consistency. This is a code style issue.